src/scripts/eastern_kingdoms/elwynn_forest/elwynn_forest.cpp

```diff
@@ -14,80 +14,149 @@
  * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
  */
 
-/* ScriptData
-SDName: Elwynn_Forest
-SD%Complete: 50
-SDComment: Quest support: 1786
-SDCategory: Elwynn Forest
-EndScriptData */
-
-/* ContentData
-npc_henze_faulk
-EndContentData */
-
 #include "scriptPCH.h"
 
-/*######
-## npc_henze_faulk
-######*/
+namespace tome_of_divinity
+{
+    // Script name: tome_of_divinity
+    // Used by two npc's
+    // https://www.wowhead.com/classic/npc=6172/henze-faulk
+    // https://www.wowhead.com/classic/npc=6177/narm-faulk
 
-#define SAY_HEAL    -1000187
+    constexpr uint32 SAY_HEAL{ 2283 };
+    constexpr uint32 SPELL_SYMBOL_OF_LIFE{ 8593 };
```

Comment:
If you are going to change macros into variables then make them static. You don't need to access them outside this cpp file. No need to export the symbol.